src/app/main.cpp

```diff
@@ -59,8 +59,9 @@ Q_IMPORT_PLUGIN(QICOPlugin)
 
 #ifdef Q_OS_UNIX
 #include <signal.h>
-#include <execinfo.h>
+#ifdef __GLIBC__
```

Comment:
I think moving this #ifdef to stacktrace.h is better, guard it around execinfo.h and also print_stacktrace() there.